### PR TITLE
{lib}[GCCcore/6.4.0] libpthread-stubs v0.4

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.4-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+"""
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('binutils','2.28'),
```
Comment (verdurin) [inline easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.4-GCCcore-6.4.0.eb]: Please fix the PEP8 violation here:
```
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.4-GCCcore-6.4.0.eb:21:16: E231 missing whitespace after ','
```

File: easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.4-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ a small footprint, latency hiding, direct access to the protocol, improved
+ threading support, and extensibility.
+"""
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['50d5686b79019ccea08bcbd7b02fe5a40634abcfd4146b6e75c6420cc170e9d9']
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.4-GCCcore-6.4.0.eb]: @JackPerdue `toolchain(opts)` is usually included above `source*`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2421.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4290de5095bbd8c1b4e1f54f4f9710d2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2133.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7b2615863cb1de08f3e62c46c875e691 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**